### DISCUSSION TITLE
Signal Strenght - how to measure?

### DISCUSSION THREAD

**Discussion by rumicagit** [merbanan/rtl_433#2076] [Q&A] [answered]
Hi guys,
Any way of measure the signal strength received to find the better spot for the antenna? Is there any option in rtl_433 syntaxe?
Regards.

**Comment by zuckschwerdt** [comment, marked as answer]
You can use `-M level` and perhaps also `-M noise` but be aware that it's dbFS depending on the current (auto) gain.
https://triq.org/rtl_433/OPERATION.html#outputs

**Comment by rumicagit** [comment]
-M level is enough! Many thanks.
